packages/cli/src/main.ts

```diff
@@ -45,9 +51,54 @@ const args = yargs(process.argv.slice(2))
         type: 'string',
         describe:
           'The contract address of the vault when `chainflip-network` is `localnet`',
+      })
+      .option('eth-network', {
+        type: 'string',
+        describe:
+          'The eth network URL to use when `chainflip-network` is `localnet`',
```

Comment:
i think this will be used anyway if it is set. should we ignore this if the `chainflip-network` is not `localnet`?

Reply:
not sure how the other arguments like `vault-contract-address` are handled

Reply:
Was going to leave this because the CLI is mostly intended for internal usage but I fixed it up.